### DISCUSSION TITLE
Copilot doesnt hides duplicates, but it says it hides duplicates.

### DISCUSSION THREAD

**Discussion by renhiyama** [community/community#10683] [Copilot Conversations]
I think instead of showing those duplicates, just hide them? Just as the title says?
![Screenshot_20220128_150018](https://user-images.githubusercontent.com/92968534/151522393-7085613a-10e4-4588-bbab-d5655791c8c9.png)

**Comment by bmuskalla** [comment]
I believe these are not duplicates (the quotation marks differ as well as the capitalization of words). I'll leave it up to the team to comment on determining equality/similarity of different solutions

**Reply by keegan-lillo** [reply]
These however are all duplicates:

<img width="613" alt="image" src="https://user-images.githubusercontent.com/3537963/162881906-6636b3ef-4a57-417b-a1b7-8005250d1201.png">

**Reply by renhiyama** [reply]
Agreed, they do are.

**Comment by renhiyama** [comment]
I believe copilot devs can make a simple code to check for 100% duplicates and hide them in results, and show that extra results generated are duplicates, the AI can't generate any new non duplicated code.
